packages/app/src/cli/validators/extensions/functions.ts

```diff
@@ -1,37 +0,0 @@
-import {FunctionExtension} from '../../models/app/extensions.js'
-import {fileExists} from '@shopify/cli-kit/node/fs'
-import {AbortError} from '@shopify/cli-kit/node/error'
-import {outputContent, outputToken, stringifyMessage} from '@shopify/cli-kit/node/output'
-
-const extensionLine = (extension: {id: string; path: string}): string => {
```

Comment:
Did you try the output before/after this removal? It seems to be different, but maybe a simpler output is what you were aiming for as well.

Reply:
It is different, but I wanted to simplify this validation. It now shows the error per extension, because it's weird that multiple functions have the same issue at the same time...
I can re-review this, but I think is the way to way, each extension defines and shows its own errors